src/app/shared/components/get-movies/get-movies.component.ts

```diff
@@ -0,0 +1,38 @@
+import { GetMoviesService } from '@shared/services/get-movies.service';
+import { Movies } from '@shared/interfaces/movies';
+import { MessageService } from '@shared/services/message.service';
+import { Router } from '@angular/router';
+// import { FavoriteMoviesService } from '@modules/user/favorite-movies/favorite-movies.service';
+import { SITEURLS } from '@shared/siteUrls';
+import { Component, OnInit, Input } from '@angular/core';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-get-movies',
```

Comment:
components like this and others in the shared folder are good candidates for changeDetection onPush

Reply:
Ok, now search, search-results, get-movies and header components are with onPush changeDetection